go-controller/pkg/util/ovs.go

```diff
@@ -328,6 +328,44 @@ func GetOVSOfPort(args ...string) (string, string, error) {
 	return stdout, stderr, err
 }
 
+func GetDatapathType(bridge string) (string, error) {
```

Comment:
The `bridge` argument is not used inside the function. 

Replace `br-int` in L332 to with `bridge`.

Reply:
ack